src/main/java/net/earthcomputer/clientcommands/command/KitCommand.java

```diff
@@ -0,0 +1,191 @@
+        saveFile();
+        sendFeedback("commands.ckit.edit.success", name);
+        return 0;
+    }
+
+    private static int load(ServerCommandSource source, String name, boolean override) throws CommandSyntaxException {
+        if (!client.player.isCreative() || !client.player.abilities.creativeMode) {
+            throw NOT_CREATIVE_EXCEPTION.create();
+        }
+
+        loadFile();
+
+        ListTag kit = kits.get(name);
+        if (kit == null) {
+            throw NOT_FOUND_EXCEPTION.create(name);
+        }
+
+        PlayerInventory tempInv = new PlayerInventory(client.player);
+        tempInv.deserialize(kit);
+        List<Slot> slots = client.player.playerScreenHandler.slots;
+        for (int i = 0; i < slots.size(); i++) {
+            if (slots.get(i).inventory == client.player.inventory) {
+                ItemStack itemStack = tempInv.getStack(((ISlot) slots.get(i)).getIndex());
+                if (!itemStack.isEmpty() || override) { // same as if (!(itemStack.isEmpty() && !override))
+                    client.interactionManager.clickCreativeStack(itemStack, i);
+                }
+            }
+        }
+
+        client.player.playerScreenHandler.sendContentUpdates();
+        sendFeedback("commands.ckit.load.success", name);
+        return 0;
+    }
+
+    private static int list(ServerCommandSource source) throws CommandSyntaxException {
+        loadFile();
+
+        String list = String.join(", ", kits.keySet());
+        sendFeedback(list.equals("") ? "No available kits" : "Available kits: " + list);
+        return kits.size();
+    }
+
+    private static void saveFile() throws CommandSyntaxException {
+        try {
+            CompoundTag rootTag = new CompoundTag();
+            CompoundTag compoundTag = new CompoundTag();
+            kits.forEach(compoundTag::put);
+            rootTag.putInt("DataVersion", SharedConstants.getGameVersion().getWorldVersion());
+            rootTag.put("Kits", compoundTag);
+            File newFile = File.createTempFile("kits", ".dat", configPath.toFile());
+            NbtIo.write(rootTag, newFile);
+            File backupFile = new File(configPath.toFile(), "kits.dat_old");
+            File currentFile = new File(configPath.toFile(), "kits.dat");
+            Util.backupAndReplace(currentFile, newFile, backupFile);
+        } catch (Exception e) {
+            throw SAVE_FAILED_EXCEPTION.create();
+        }
+    }
+
+    private static void loadFile() throws CommandSyntaxException {
+        try {
+            kits.clear();
+            CompoundTag rootTag = NbtIo.read(new File(configPath.toFile(), "kits.dat"));
+            if (rootTag == null) {
+                return;
+            }
+            final int version = SharedConstants.getGameVersion().getWorldVersion();
+            CompoundTag compoundTag = rootTag.getCompound("Kits");
+            if (rootTag.getInt("DataVersion") == version) {
```

Comment:
Could extract `rootTag.getInt("DataVersion")` to a variable. Also, this should be `>=` in case the kits file was saved in a future version.